src/moj/components/banner/template.njk

```diff
@@ -1,4 +1,26 @@
-<div class="moj-banner{% if params.type == 'success' %} moj-banner--success{% elif params.type == 'warning' %} moj-banner--warning{% else %} {{- ' ' + params.classes if params.classes}}{% endif %}" {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %} role="region" aria-label="{{ params.iconFallbackText | default(params.type) | default("banner") }}">
+{#- Set classes for this component #}
+{%- set classNames = ["moj-banner"] -%}
+
+{%- if params.type == 'success' %}
+  {%- set classNames = classNames.concat("moj-banner--success") %}
+{%- elif params.type == 'warning' %}
+  {%- set classNames = classNames.concat("moj-banner--warning") %}
+{%- elif params.type == 'information' %}
+  {%- set classNames = classNames.concat("moj-banner--information") %}
+{%- endif %}
+
+{%- set classNames = classNames.concat(params.classes) if params.classes %}
+
+{%- set classNames = classNames.join(' ') -%}
```

Comment:
This is causing the build to fail -- I'm looking into this, but might revert the refactor if I cannot find an immediate soluion (to fix the bug as a priority)